authcontroller.py

```diff
@@ -605,20 +611,6 @@ def confirm_organisation(token=None):
         Please request these from ORCID by clicking on link 'Take me to ORCID to obtain Client iD and Client Secret'
         and come back to this form once you have them.""", "warning")
 
-        redirect_uri = url_for("orcid_callback", _external=True)
-        clientSecret_url = iri_to_uri(MEMBER_API_FORM_BASE_URL) + "?" + urlencode(
-            dict(
-                new_existing=NEW_CREDENTIALS,
-                note=NOTE_ORCID + " " + user.organisation.name,
-                contact_email=email,
-                contact_name=user.name,
-                org_name=user.organisation.name,
-                cred_type=CRED_TYPE_PREMIUM,
-                app_name=APP_NAME + " at " + user.organisation.name,
-                app_description=APP_DESCRIPTION + " at " + user.organisation.name,
-                app_url=APP_URL,
-                redirect_uri_1=redirect_uri))
-
         try:
             orgInfo = OrgInfo.get(email=email)
```

Comment:
we could retrieve the org info either by tech.contact email or the supplied organisation name:
```OrgInfo.email == email or OrgInfo.name == org_name or OrgInfo.tuakiri_name == org_name ```

where org_name would be either users curent organisation name or the name kept in the token (see: notes above)